documentation/docs/contribute.md

```diff
@@ -9,35 +9,36 @@ keywords:
 - Identity Initiative
 - Documentation
 - Discord
+- reference
 ---
 
-**Thanks for thinking about contributing to the project! We have the following ways that you can contribute.**
+**Thanks for thinking about contributing to the project! You can contribute using the following methods.**
 
 ## Join the Identity Initiative
 
-The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience.
+The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience that includes:
 
-- Quality assurance and review
-- Documentation
-- Code samples
+- Quality assurance and review.
+- Documentation.
+- Code samples.
 
-If you'd like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
+If you would like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
 
-## Contribute to the project's GitHub repository
+## Contribute to the Project's GitHub Repository
 
-All the code is open source and hosted on [GitHub](https://github.com/iotaledger/identity.rs) where you can do the following:
+All of the code is open source and hosted on [GitHub](https://github.com/iotaledger/identity.rs) where you can:
 
-- Report a bug
-- Suggest a new feature
-- Contribute to the documentation
+- Report a bug.
+- Suggest a new feature.
+- Contribute to the documentation.
 
-## Contribute to the documentation
+## Contribute to the Documentation
 
 This documentation is also open source and hosted on GitHub.
 
 If you want to contribute new documentation or fix an error, see the [contribution guidelines](https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md).
```

Comment:
"This repository has been archived by the owner. It is now read-only."  Perhaps we could update these links to use the wiki, or update https://github.com/iotaledger/identity.rs/blob/dev/.github/CONTRIBUTING.md and use that.  @Dr-Electron had been working on a base repo so we can unify this sort of files.

Reply:
Works for me👌